pkg/flowkit/contracts/contracts.go

```diff
@@ -39,6 +39,7 @@ type Contract struct {
 	name         string
 	source       string
 	target       flow.Address
```

Comment:
change this to `accountAddress` or change `accountName` to `targetName` either way as long as they are consistent.

Reply:
addressed